src/cli/index.ts

```diff
@@ -0,0 +1,59 @@
+// NOTE :: All commands are stubbed as examples
+// Useful repo https://github.com/tsantef/commander-starter
+const program = require("commander");
```

Comment:
linter wants `import program from "commander";` format

Reply:
yeah i was getting all sort of weird things with this import... will work on it